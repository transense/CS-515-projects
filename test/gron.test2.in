{
    "Name": "stevens",
    "category": "Educational",
    "location": "hoboken",
    "contacts": {
      "email": "example.com",
      "phone": "123-456-789"
    },
    "hobbies": ["reading", "traveling"]
  }
  