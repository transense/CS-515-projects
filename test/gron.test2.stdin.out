json = {};
json.Name = "stevens";
json.category = "Educational";
json.contacts = {};
json.contacts.email = "example.com";
json.contacts.phone = "123-456-789";
json.hobbies = [];
json.hobbies[0] = "reading";
json.hobbies[1] = "traveling";
json.location = "hoboken";

